### PR TITLE
docs: edits to zoom vignette

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: vignettes/tech-dm-zoom.Rmd [hunk 1]
```diff
@@ -60,9 +60,8 @@ flights_zoomed
 
 flights_zoomed_mutate <-
   flights_zoomed %>%
-  mutate(am_pm_dep = if_else(dep_time < 1200, "am", "pm")) %>%
-  # in order to see our changes in the output we use `select()` for reordering the columns
-  select(year:dep_time, am_pm_dep, everything())
+  mutate(am_pm_dep = if_else(dep_time < 1200, "am", "pm"), .after = "dep_time")
```
Comment (krlmlr) [inline vignettes/tech-dm-zoom.Rmd:63]: This isn't going to work yet with zooming, #1022.

**Review by krlmlr — COMMENTED** [review]
Thanks, some API features are only wishful thinking for now...

File: vignettes/tech-dm-zoom.Rmd (hunk 1 above)
Comment (IndrajeetPatil) [reply vignettes/tech-dm-zoom.Rmd]: Ah, I see. I've reverted this change.

**Comment by IndrajeetPatil** [comment]
@krlmlr  This should be ready for a merge. No pending review comments.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**